android/app/src/main/java/com/microsoft/codepush/react/CodePushDialog.java

```diff
@@ -50,12 +67,20 @@ public void onHostDestroy() {
         }
     }
 
+    /**
+     * Internal method for actually showing the dialog.
+     *
+     * @param title           title of the dialog.
+     * @param message         message to be displayed.
+     * @param button1Text     text on the left button.
```

Comment:
please avoid of using `button1Text ` and `button2Text ` names, please provide exact naves what buttons are doing.